Unify and change of segmentation/surface syntax

dkuegler opened the pull request:
Previous commits and versions have often used a --flag (seg|surf)=value syntax. This Syntax is replaced by --flag_(seg|surf) value. This specifically affects brun_fastsurfer.sh and srun_fastsurfer.sh, which had these Syntaxes established.

This PR still needs extensive testing.

Also, not all doc entries are updated (specifically `doc/scripts/BATCH.md` and `doc/scripts/SLURM.md`).

dkuegler committed:
Unify and change of segmentation/surface syntax

Previous commits and versions have often used a --flag (seg|surf)=value syntax. This Syntax is replaced by --flag_(seg|surf) value.
This specifically affects brun_fastsurfer.sh and srun_fastsurfer.sh, which had these Syntaxes established.

m-reuter commented on brun_fastsurfer.sh:
```diff
@@ -67,24 +67,25 @@ iii. a list of subjects directly passed (this does not support subject-specific
-    --parallel_subjects on gpus, calculate the "safe" number of concurrent segmentations and
-    specify "<m>" with the flag '--parallel_subjects seg=<m>' in addition to
-    '--parallel_subjects surf=<n>' (together max. m+n processes will run, n/m may be "max").
+--parallel_subjects <n>|max: parallel execution of run_fastsurfer for <n> images. Creates <n>
```
can we just call this parallel and parallel_seg ... ? Or is that conflicting with the parallel flag from the surf run (for hemispheres)?

dkuegler replied:
It is conflicting with the `--parallel` flag, which also is a valid flag here.

dkuegler replied:
One other option could be `--num_processes` but I am not sure this would not get confused with `--threads` and the like.
Unfortunately, `brun_fastsurfer.sh` really has 3+ different parallel processing options that have interaction effects:
`--threads`, `--parallel`, and `--parallel_subjects*`.

dkuegler committed:
Update the documnetation for BATCH processing (naming --parallel_subjects_seg instead of --parallel_seg)

dkuegler commented:
Fixed the documentation.

dkuegler committed:
Removing extra debug output from brun_fastsurfer.sh

Remove set -x and set +x from brun (which were there for debugging)
Remove job pid printing
fix IFS for better output

dkuegler commented:
Tested and fixed srun_fastsurfer and brun_fastsurfer
Fixes are only debug outputs
